sklearn/ensemble/voting_classifier.py

```diff
@@ -127,6 +128,12 @@ def fit(self, X, y):
 
         return self
 
+    def _check_fitted(self):
+        """ Raises NotFittedError if object was not fitted"""
+        if not hasattr(self, 'estimators_') or self.estimators_ is None:
```

Comment:
Good point, I extended the test to also check for empty lists, e.g., self.estimators_=[]

> On Nov 12, 2015, at 12:14 PM, Manoj Kumar <notifications@github.com> wrote:
> 
> In sklearn/ensemble/voting_classifier.py <https://github.com/scikit-learn/scikit-learn/pull/5794#discussion_r44685087>:
> 
> > @@ -127,6 +128,12 @@ def fit(self, X, y):
> >  
> >          return self
> >  
> > +    def _check_fitted(self):
> > +        """ Raises NotFittedError if object was not fitted"""
> > +        if not hasattr(self, 'estimators_') or self.estimators_ is None:
> @rasbt <https://github.com/rasbt> could you also verify that len(self.estimators) is greater than 0?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/scikit-learn/scikit-learn/pull/5794/files#r44685087>.
>